tests/unit/Hooks/ImageHooksTest.php

```diff
@@ -112,7 +117,7 @@ public function testPostCreate() {
 			->method('insert')
 			->with($this->imageDimension);
 
-		$this->imagehooks->post_create($jpgFile);
+		$this->mockImageHooks->postCreate($jpgFile);
```

Comment:
Something must have changed in your PHP env. Try maybe to use PHP 7?
If it seg faults here, chances are it will seg fault as well when using the app